inventory_system.py:346:1: W391 blank line at end of file
